modules/Microsoft.ContainerInstance/containerGroups/deploy.bicep

```diff
@@ -49,6 +102,46 @@ var identity = identityType != 'None' ? {
   userAssignedIdentities: !empty(userAssignedIdentities) ? userAssignedIdentities : null
 } : null
 
+var dnsConfig = !empty(dnsNameServers) ? {
+  nameServers: dnsNameServers
+  searchDomains: dnsSearchDomains
+} : null
+
+var encryptionProperties = !empty(encryptionVaultBaseUrl) ? {
+  vaultBaseUrl: encryptionVaultBaseUrl
+  keyName: encrytionKeyName
+  keyVersion: encryptionKeyVersion
+} : null
+
+var subnetIds = !empty(subnetId) ? [
+  {
+    id: subnetId
```

Comment:
Question: As this is an array, should we allow providing mutliple subnetResourceIds - or is that actually not supported?

Reply:
not supported ;)